docs/notebooks/bsync_examples/example1.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<BuildingSync xmlns="http://buildingsync.net/schemas/bedes-auc/2019" version="2.3.0">
+<BuildingSync xmlns="http://buildingsync.net/schemas/bedes-auc/2019" version="2.4.0">
```

Comment:
after fixing and running the entire notebook, this was the only change to the example file, which means that all is still right with the world.

Reply:
phew